src/test/java/com/defano/hypertalk/ast/expression/function/VersionFuncTest.java

```diff
@@ -0,0 +1,40 @@
+package com.defano.hypertalk.ast.expression.function;
+
+import com.defano.hypertalk.GuiceTest;
+import com.defano.hypertalk.ast.expression.Expression;
+import com.defano.hypertalk.ast.model.Value;
+import com.defano.hypertalk.exception.HtSemanticException;
+import com.google.common.collect.Lists;
+import org.junit.Ignore;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class VersionFuncTest extends GuiceTest<VersionFunc> {
+
+    private Expression mockExpression = Mockito.mock(Expression.class);
+
+    @BeforeEach
+    public void setUp() {
+
+        initialize(new VersionFunc(mockParserRuleContext));
+    }
+
+    @Test
+    public void testNoArguments() throws Exception {
```

Comment:
You might change this to `testTheVersion` since the test report will identify this test by the method name and this method name is pretty generic.

Reply:
Sure. Please see
![image](https://user-images.githubusercontent.com/3395566/169669275-2044f431-c8db-4ee2-bc65-9b055b7ffe70.png)